packages/common/package.json

```diff
@@ -2,13 +2,21 @@
   "name": "@graduate/common",
   "version": "1.0.0",
   "main": "dist/index.js",
+  "module": "module/index.js",
+  "browser": "lib/index.js",
   "private": true,
   "scripts": {
-    "dev": "yarn build -w",
-    "build": "babel --root-mode upward -d dist -x '.ts' *.ts"
+    "dev": "yarn build:dev -w",
+    "build": "yarn build:dev && yarn build:browser && yarn build:module",
+    "build:dev": "yarn g:babel --root-mode upward -d dist -x '.ts' *.ts",
+    "build:browser": "yarn g:cross-env BABEL_ENV=browser yarn g:babel --root-mode upward -d lib -x '.ts' *.ts",
+    "build:module": "yarn g:cross-env BABEL_ENV=module yarn g:babel --root-mode upward -d module -x '.ts' *.ts"
```

Comment:
build the common package 3 different ways into dist/ browser/ module/. dist/ is used in dev, browser/ is used by frontend application and module/ is used by NodeJS applications(api-v2)